tools/cabana/mainwin.cc

```diff
@@ -246,6 +246,11 @@ void MainWindow::loadDBCFromClipboard() {
 }
 
 void MainWindow::loadDBCFromFingerprint() {
+  // Don't overwrite already loaded DBC
```

Comment:
Need to test if this still works under Linux. on MacOS loading from fingerprint is broken.

Reply:
Still works fine!